crates/matrix-sdk-crypto/src/machine.rs

```diff
@@ -862,24 +861,26 @@ impl OlmMachine {
         #[derive(Deserialize)]
         struct ContentStub<'a> {
             #[serde(borrow, rename = "org.matrix.msgid")]
-            message_id: JsOption<&'a str>,
+            message_id: Option<&'a str>,
         }
 
         #[derive(Deserialize)]
         struct ToDeviceStub<'a> {
+            sender: &'a str,
+            #[serde(rename = "type")]
+            event_type: &'a str,
             #[serde(borrow)]
             content: ContentStub<'a>,
         }
 
-        let message_id = event
-            .deserialize_as::<ToDeviceStub<'_>>()
-            .ok()
-            .and_then(|t| t.content.message_id.into_option());
-
-        tracing::Span::current().record("message_id", message_id);
+        if let Ok(event) = event.deserialize_as::<ToDeviceStub<'_>>() {
+            tracing::Span::current().record("sender", event.sender);
```

Comment:
IIRC using `field::display` or `%` inside the macros stops tracing from adding quotes around the string or escaping things inside. Do you prefer the quotes? (we are kinda inconsistent about it)

Reply:
Trying this out, I think I prefer them, especially since some things may be empty strings.

Reply:
Right. Maybe we should use debug representation for user IDs and such everywhere then, which IIRC print the quoted string.